scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -1,24 +1,34 @@
 #!/bin/bash -xe
 set +o pipefail
 
-python -m pip install --upgrade pip docker-hub==2.2.0
-echo docker-hub > dev-requirements.txt
+install_ci_dependencies (){
+  python -m pip install --upgrade pip
+  echo docker-hub==2.2.0 > dev-requirements.txt
+  pip install -r dev-requirements.txt
+}
+
+get_driver_version (){
+  source /home/runner/.bash_profile
+  cd common
+  image_version=`yq eval .identity.version config.yaml`
+  cd -
+}
 
+install_ci_dependencies
 cat >>/home/runner/.bash_profile <<'EOL'
 yq() {
-  docker run --rm -e operator_image_for_test=$operator_image_for_test\
-                  -e cr_image_value=$cr_image_value\
-                  -i -v "${PWD}":/workdir mikefarah/yq "$@"
+  docker run --rm -i -v "${PWD}":/workdir mikefarah/yq "$@"
 }
 EOL
+get_driver_version
+GITHUB_SHA=${GITHUB_SHA:0:7}_
```

Comment:
What does `GITHUB_SHA` mean?

Reply:
this is the commit hash/sha

Reply:
this is a built-in variable in the github actions

Reply:
Then I suggest renaming it to 
```suggestion
COMMIT_HASH=${COMMIT_HASH:0:7}_
```

Reply:
I can only do like this:
```suggestion
COMMIT_HASH=${GITHUB_SHA:0:7}_
```
because the `${GITHUB_SHA}` parameter is built-in parameter in GitHub actions

Reply:
In this case I think we can keep it